dash/dash.py

```diff
@@ -754,15 +755,47 @@ def _validate_callback(self, output, inputs, state, events):
             ).replace('    ', ''))
 
         callback_id = _create_callback_id(output)
-        if callback_id in self.callback_map:
-            raise exceptions.CantHaveMultipleOutputs('''
+        is_multi = isinstance(output, (list, tuple))
+        callbacks = set(itertools.chain(*(
+            x[1:-1].split(':')
+            if x.startswith('[')
```

Comment:
Backwards incompatible with apps that use a callback output ID starting with '['